README.md

```diff
@@ -295,29 +295,221 @@ Se asume que la persona tiene conocimientos previos en:
+
+ 1. Ahora ejecutemos la prueba. Desde la clase RestAssuredAuth.java ejecuta la prueba, verifica que el test quedo OK.
+ 
+ 1. Has fallar tu asercion, en el statusCode(200), cambialo por 300 y ejecuta nuevamente. Veras que ahora la prueba quedo fallida.
+ 
+ Puedes ver más Matchers [Aquí]( https://www.javadoc.io/doc/org.hamcrest/hamcrest/2.1/org/hamcrest/Matchers.html).
+ 
+ 
+  ### 5. Configuremos nuestro reporte con Allure
+  
+ 1. En el archivo pom.xml agrega la dependencia de Allure que se encuentra en el repositorio de mavem.
+ 
+	 Copie y pegue:
+	 ```java
+	 <dependency>
+		<groupId>io.qameta.allure</groupId>
+		<artifactId>allure-testng</artifactId>
+		<version>2.18.1</version>
+		<scope>test</scope>
+	</dependency>
+	```
+	
+ 1. En el archivo pom.xml agregue en la seccion properties la siguiente linea
+ 
+	 Copie y pegue:
+	 ```java
+	 <aspectj.version>1.8.10</aspectj.version>
+	 ```
+	    
+ 1. En el archivo pom.xml agrega los siguientes plogins que le permitiran generar el reporte. Recuerde que en 
+ 
+	 Copie y pegue:
+	 ```java
+	 <plugin>
+	    <groupId>org.apache.maven.plugins</groupId>
+		<artifactId>maven-compiler-plugin</artifactId>
+		<configuration>
+			<source>1.8</source>
+			<target>1.8</target>
+		</configuration>
+	 </plugin>
+	 <plugin>
+		<groupId>org.apache.maven.plugins</groupId>
+		<artifactId>maven-surefire-plugin</artifactId>
+		<version>2.20</version>
+		<configuration>
+			<argLine>
+				-javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"
+			</argLine>
+		</configuration>
+		<dependencies>
+			<dependency>
+				<groupId>org.aspectj</groupId>
+				<artifactId>aspectjweaver</artifactId>
+				<version>${aspectj.version}</version>
+			</dependency>
+		</dependencies>
+		</plugin>
+		<plugin>
+			<groupId>io.qameta.allure</groupId>
+			<artifactId>allure-maven</artifactId>
+			<version>2.8</version>
+			<configuration>
+				<reportVersion>2.7.0</reportVersion>
+				<allureDownloadUrl>https://github.com/allure-framework/allure2/releases/download/2.7.0/allure-2.7.0.zip</allureDownloadUrl>
+				<resultsDirectory> ${basedir}\allure-results</resultsDirectory>
+			</configuration>
+		</plugin>
+	 ```
+	 
+ 1. Para que los cambios sean tomados actualice las librerías. Desde Eclipse puede hacer clic derecho desde el proyecto, seleccione la opción Maven y luego Update Project. Verifique que este seleccionado el proyecto sobre el cual esta trabajando y luego ejecute OK.
+ 
+ 1. Ahora actialice el metodo "test1" de la clase RestAssuredAuth.java para que quede de la siguiente forma.
+ 
+ 	Copie y pegue:
+	```java
+	@Test(priority = 0, description="Valid Autentication Scenario with valid username and password.")
+	@Severity(SeverityLevel.BLOCKER)
+	@Description("Test Description: Login test with valid username and password.")
+	@Story("Get autentication token")
+	@Step("Petition get to autentication")
```

Comment:
Is necessary?:

- priority, description into @Test
- @Severity
- @Story
- @Step